image/image.go

```diff
@@ -223,6 +223,52 @@ func (i Ref) WithNewTag(t string) Ref {
 	return img
 }
 
+// Labels has all the image labels we are interested in for an image
+// ref, the JSON struct tag keys should be equal to the label.
+type Labels struct {
```

Comment:
No strong opinion, but wouldn't it be more flexible to make this a map? (in case we end up adding more labels in the future).

I guess we can always make it a map in the future without breaking the JSON representation, right?

Reply:
> I guess we can always make it a map in the future without breaking the JSON representation, right?

This is just a JSON object with key/values, so yeap, no worries.

Reply:
Makes sense